tests/app/preheat_multiple_notebooks_test.py

```diff
@@ -4,7 +4,7 @@
 import os
 
 BASE_DIR = os.path.dirname(__file__)
-NOTEBOOK_EXECUTION_TIME = 3
+NOTEBOOK_EXECUTION_TIME = 2
```

Comment:
```suggestion
NOTEBOOK_EXECUTION_TIME = 3
```
It is not related to `ipywidgets`, I added it to increase CI robustness.

Reply:
Thanks!